app/views/shared/_cancel_action_modal.html.slim

```diff
@@ -0,0 +1,30 @@
+- app_flow = idv ? 'idv' : 'sign_up'
+
+#cancel-action-modal.display-none
+  = render layout: 'shared/modal_layout' do
+    .mx-auto.p5.cntnr-xskinny.border-box.bg-white.rounded.relative.rounded-lg
+      .modal-warning
```

Comment:
I _think_ with that new modal partial, this `.modal-warning` needs to be applied on the div above it, not on it's own.  Also the `.mx-auto` isn't needed since that is already being applied in the stylesheet.